SDK/AppCenter/Microsoft.AppCenter.Shared/AppCenter.cs

```diff
@@ -11,50 +13,66 @@ namespace Microsoft.AppCenter
     /// </summary>
     public partial class AppCenter
     {
-        // Gets the first instance of an app secret corresponding to the given platform name, or returns the string 
+        const string SecretDelimiter = ";";
+        const string PlatformKeyValueDelimiter = "=";
+        const string TargetPostfix = "target";
```

Comment:
Given the format of `"appSecret=123;target=123"`, I don't think they are "postfix" at all. 

Maybe something like
```
        const string TargetKeyName = "target";
        const string AppSecretKeyName = "appsecret";
```